ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -258,6 +258,7 @@
       "CONFIG_whisk_containerFactory_containerArgs_network": "{{ invoker_container_network_name | default('bridge') }}"
       "INVOKER_CONTAINER_POLICY": "{{ invoker_container_policy_name | default()}}"
       "CONFIG_whisk_containerPool_userMemory": "{{ hostvars[groups['invokers'][invoker_index | int]].user_memory | default(invoker.userMemory) }}"
+      "CONFIG_whisk_containerPool_userCpus": "{{ invoker.userCpus | default() }}"
```

Comment:
I could deploy this branch w/ and w/o this configuration using ansible.
I confirmed that NanoCpus is properly configured in containers according to this config.

```json
...
            "CpuShares": 25,
            "Memory": 268435456,
            "NanoCpus": 0,
...
```

```json
...
            "CpuShares": 25,
            "Memory": 268435456,
            "NanoCpus": 150000000,
...
```